pkg/controllers/management/auth/project_cluster_handler.go

```diff
@@ -54,7 +56,7 @@ func newPandCLifecycles(management *config.ManagementContext) (*projectLifecycle
 		crtbLister:           management.Management.ClusterRoleTemplateBindings("").Controller().Lister(),
 		crtbClient:           management.Management.ClusterRoleTemplateBindings(""),
 		projectLister:        management.Management.Projects("").Controller().Lister(),
-		projects:             management.Management.Projects(""),
+		projects:             management.Wrangler.Mgmt.Project(),
```

Comment:
Just curious - why is this changing (and also switching from `v3.ProjectInterface` to `wranglerv3.ProjectClient` below?

Reply:
This is needed to bypass the webhook, which can't be done with the Norman client